plotly/offline/offline.py

```diff
@@ -185,3 +165,12 @@ def plot():
     """ Configured to work with localhost Plotly graph viewer
     """
     raise NotImplementedError
+
+
+try:
+    _init_notebook_mode()
```

Comment:
and a funny caveat is that now this bundle will be shared by anyone who has offline mode, regardless of whether they made an offline graph or not!